### DISCUSSION TITLE
Obter o valor de um p-options de um PO-COMBO

### DISCUSSION THREAD

**Discussion by FernandoHinode** [po-ui/po-angular#1591] [Tire suas dúvidas]
Bom Dia,

Prezados, criei uma DIV que tem um PO-COMBO e passo o p-options, gostaria de fazer uma função pra pegar o texto em que o usuário posicionou, exemplo se ele selecionou máquina parada, ou máquina sem ferramenta e isso seja retornado pra mim.

Como posso fazer isso?

            <div>
                <po-combo name="combo" p-label="Opções de Parada"  [p-options]="[{ value: 'Máquina Parada' }, { value: 'Máquina sem Ferramenta' }, { value: 'Máquina Quebrada' }, { value: 'Máquina Sem Óleo' }]"> </po-combo>
            </div>

**Comment by jcorrea97** [comment]
@FernandoHinode você pode utilizar [(ngModel)] ou o evento (p-change) do componente. Deixei um exemplo utilizando (p-change) no link abaixo: 
https://stackblitz.com/edit/po-ui-o4wlr2?file=src/app/app.component.ts

Caso tenha dúvida de como funciona o po-combo temos exemplos de uso na documentação: 
https://po-ui.io/documentation/po-combo

**Reply by FernandoHinode** [reply]
@jcorrea97 Muito Obrigado pela resposta, vou tentar fazer.

Abraços.
